pkg/aspect/common.go

```diff
@@ -21,10 +21,98 @@ import (
 
 	dpb "istio.io/api/mixer/v1/config/descriptor"
 	"istio.io/mixer/pkg/adapter"
+	"istio.io/mixer/pkg/adapter/config"
 	"istio.io/mixer/pkg/attribute"
+	cfg "istio.io/mixer/pkg/config"
 	"istio.io/mixer/pkg/expr"
 )
 
+// FromHandler creates a CreateAspectFunc from the provided handler instance.
+func FromHandler(handler config.Handler) CreateAspectFunc {
+	return func(adapter.Env, adapter.Config, ...interface{}) (adapter.Aspect, error) {
```

Comment:
This works because the adapter.handler interface and the adapter.Aspect interface are identical right. Once we have things working, we can work on cleaning all of that up.